authors/apps/authentication/tests.py

```diff
@@ -0,0 +1,103 @@
+# tests for authentication application
+from django.test import TestCase
+from authors.apps.authentication.models import User, UserManager
+from rest_framework.test import APIClient
```

Comment:
kindly sort these imports. starting with third-party library imports and then imports from your code.

Right clicking in the file in vscode and choosing `Sort Imports` can come in handy

Reply:
Thanks, I shall work on that

Reply:
The imports have been sorted